packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx

```diff
@@ -28,11 +29,19 @@ export const OrganizationMembers = withCardStateProvider(() => {
   const card = useCardState();
   const canManageMemberships = useProtect({ permission: 'org:sys_memberships:manage' });
   const canReadMemberships = useProtect({ permission: 'org:sys_memberships:read' });
+  const { t } = useLocalizations();
   const isDomainsEnabled = organizationSettings?.domains?.enabled;
-  const { membershipRequests } = useOrganization({
+  const { membershipRequests, memberships, invitations } = useOrganization({
     membershipRequests: isDomainsEnabled || undefined,
+    invitations: true,
```

Comment:
🤔 Maybe we should check permissions here instead of defaulting to true. And the same goes for `membershipRequests` so we may have a bug there.

Reply:
```suggestion
    invitations: canManageMemberships,
```

Reply:
Thank you for bringing that up!